.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,62 @@
+name: Pull Request Checks 
+
+on:
+  pull_request:
+    types: [opened]
+  pull_request_review:
+    types: [submitted]
+
+concurrency: pr-${{ github.ref }}
+
+jobs:
+  dependency-checks:
+    name: Dependency Checks
+    # run if the PR was opened or there was a comment containing '/pr_checks'
+    if: (github.event_name == 'pull_request') || (github.event_name == 'pull_request_review' && contains(github.event.review.body, '/pr_checks'))
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Checkout Code
+      uses: actions/checkout@v2        
+
+    - name: Set Branch Env Var
+      run: | 
+        echo "BRANCH_NAME=$(echo ${{github.event.pull_request.head.ref}} | tr / -)" >> $GITHUB_ENV
+
+    - name: Set up Java8
+      uses: actions/setup-java@v2
+      with:
+        java-version: '8'
+        distribution: 'adopt-hotspot'
+        cache: 'gradle'      
+
+    - name: Gradle Publish To MavenLocal
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        ./gradlew clean build publishToMavenLocal --info
```

Comment:
Minor nit: do we need `--info`?  Same for line 46.

Reply:
Good point, I removed them from both line 37 and line 46.